airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalClient.java

```diff
@@ -426,16 +427,7 @@ public ManualOperationResult synchronousResetConnection(final UUID connectionId,
       return resetResult;
     }
 
-    try {
-      ConnectionManagerUtils.getConnectionManagerWorkflow(client, connectionId);
-    } catch (final Exception e) {
-      log.error("Encountered exception retrieving workflow after reset.", e);
-      return new ManualOperationResult(
-          Optional.of(e.getMessage()),
-          Optional.empty());
-    }
```

Comment:
Removed this block since I don't think it is doing anything useful anymore (this was only used when we were saving the result of `getConnectionManagerWorkflow` in the past)